pkg/locodedb/calls.go

```diff
@@ -23,7 +24,8 @@ func Get(locodeStr string) (Record, error) {
 	if err := initLocodeData(); err != nil {
 		return Record{}, err
 	}
-
+	locodeStr = strings.ReplaceAll(locodeStr, " ", "")
+	locodeStr = strings.ReplaceAll(locodeStr, "-", "")
```

Comment:
1. It's a completely separate change -> different commit at least.
2. It's not entirely correct, I can pass `   XX---YYY - - - -` this way.
3. It's suboptimal, you don't need additional allocations here, just split the string properly depending on the input (that is either a 5 or 6 chars).

Reply:
opened different pr